### PR TITLE
Linkaxes

### PR DESCRIPTION
**Pull request by acfloria**
- Link the axes of different figures

- Update the README with installation info and how to add a new figure

- Add a plot of the estimated position vs gps position

### CODE REVIEW

File: logconv.m [hunk 1]
```diff
@@ -23,7 +23,7 @@ function logconv()
 % ************************************************************************
 
 % set the path to your log file here file here
-fileName = 'log001.ulg';
```
Comment (ASM3) [inline logconv.m]: should be changed back to the default value

File: logconv.m [hunk 2]
```diff
@@ -33,7 +33,7 @@ function logconv()
 % 2: import the data from the .mat file
 %    (requires the generated .mat file)
 % else: Defaults to 0
-loadingMode = 0;
+loadingMode = 2;
```
Comment (ASM3) [inline logconv.m]: should be changed back to the default value

File: logconv.m [hunk 3]
```diff
@@ -61,8 +61,8 @@ function logconv()
 
 % only plot the logged data from t_start to t_end. If both are set to 0.0
 % all the logged data is plotted.
-t_start = 0.0;
-t_end = 0.0;
+t_start = 640.0;
```
Comment (ASM3) [inline logconv.m]: should be changed back to the default value

File: logconv.m [hunk 4]
```diff
@@ -61,8 +61,8 @@ function logconv()
...
+t_end = 650.0;
```
Comment (ASM3) [inline logconv.m]: should be changed back to the default value

File: logconv.m [hunk 5]
```diff
@@ -23,7 +23,7 @@ function logconv()
 % ************************************************************************
 
 % set the path to your log file here file here
-fileName = 'log001.ulg';
+fileName = '13_35_39.ulg';
```
Comment (ASM3) [inline logconv.m]: should be changed back to the default value

File: logconv.m (hunk 1 above)
Comment (acfloria) [reply logconv.m]: Already changed back

File: logconv.m (hunk 5 above)
Comment (acfloria) [reply logconv.m]: Already changed back

File: logconv.m (hunk 2 above)
Comment (acfloria) [reply logconv.m]: Already changed back

File: logconv.m (hunk 3 above)
Comment (acfloria) [reply logconv.m]: Already changed back

File: logconv.m (hunk 4 above)
Comment (acfloria) [reply logconv.m]: Already changed back

**Comment by acfloria** [comment]
@ASM3 good to merge?

**Comment by ASM3** [comment]
yes

**Pull request merged**